src/foundry/foundry.js/clientDocuments/actor.d.ts

```diff
@@ -25,10 +27,267 @@ declare global {
+     * If the Actor is a synthetic token actor, only the exact Token which it represents will be returned.
+     *
+     * @param linked   - Limit results to Tokens which are linked to the Actor. Otherwise return all Tokens even those which are not linked. (default: `false`)
+     * @param document - Return the Document instance rather than the PlaceableObject (default: `false`)
+     * @returns An array of Token instances in the current Scene which reference this Actor.
+     * TODO: Configurable token class
+     */
+    getActiveTokens(linked?: boolean, document?: boolean): Token[];
+
+    /**
+     * Prepare a data object which defines the data schema used by dice roll commands against this Actor
+     */
+    getRollData(): this['data']['data'];
+
+    /** {@inheritdoc} */
+    protected _getSheetClass(): ConstructorOf<FormApplication>;
+
+    /**
+     * Create a new TokenData object which can be used to create a Token representation of the Actor.
+     * @param data - Additional data, such as x, y, rotation, etc. for the created token data (default: `{}`)
+     * @returns The created TokenData instance
+     * TODO: TokenData does not yet exist
+     */
+    getTokenData(data?: object): Promise<object>;
+
+    /**
+     * Get an Array of Token images which could represent this Actor
+     */
+    getTokenImages(): Promise<string[]>;
+
+    /**
+     * Handle how changes to a Token attribute bar are applied to the Actor.
+     * This allows for game systems to override this behavior and deploy special logic.
+     * @param attribute - The attribute path
+     * @param value     - The target attribute value
+     * @param isDelta   - Whether the number represents a relative change (true) or an absolute change (false) (default: `false`)
+     * @param isBar     - Whether the new value is part of an attribute bar, or just a direct value (default: `true`)
+     * @returns The updated Actor document
+     */
+    modifyTokenAttribute(attribute: string, value: number, isDelta: boolean, isBar: boolean): Promise<this>;
+
+    /** {@inheritdoc} */
+    prepareEmbeddedEntities(): void;
+
+    /**
+     * Roll initiative for all Combatants in the currently active Combat encounter which are associated with this Actor.
+     * If viewing a full Actor entity, all Tokens which map to that actor will be targeted for initiative rolls.
+     * If viewing a synthetic Token actor, only that particular Token will be targeted for an initiative roll.
+     *
+     * @param options - Configuration for how initiative for this Actor is rolled.
+     * @returns A promise which resolves to the Combat entity once rolls are complete.
+     */
+    rollInitiative(options?: Actor.RollInitiativeOptions): Promise<void>;
+
+    /** {@inheritdoc} */
+    getEmbeddedCollection(embeddedName: string): Collection<foundry.abstract.Document<any, this>>;
+
+    /**
+     * {@inheritdoc}
```

Comment:
please remove `/** {@inheritdoc} */` and `@override`

Also it should be `protected`, same for the other methods starting with `_`